### PR TITLE
Adds retrospectives format to the Made Tech team

### PR DESCRIPTION
**Pull request by craigjbass**
# Retrospectives

Please help us improve them, they affect everyone!

### CODE REVIEW

File: team-norms/retrospectives.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Purpose 
+
+The purpose of retrospectives is to:
+
+* Provide a conduit to allow continuous improvement to be facilitated safely within teams.
+* Examine outcomes from previous retrospective, and understand what went well, and what didn’t.
+
+The purpose of the comradrospective is to:
+
+*  Examine previous action points, what went well and what didn’t.
```
Comment (alex-min) [inline team-norms/retrospectives.md]: could we add here "Examine previous action points [from the retrospective]" to make it more obvious?

File: team-norms/retrospectives.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Retrospective (smaller groups performing retrospectives)
+* Comradrospective (whole company retrospective format)
+
+## Purpose 
+
+The purpose of retrospectives is to:
+
+* Provide a conduit to allow continuous improvement to be facilitated safely within teams.
+* Examine outcomes from previous retrospective, and understand what went well, and what didn’t.
+
```
Comment (alex-min) [inline team-norms/retrospectives.md]: Could we add here that we create action points from the result of that retrospective?

Comment (alex-min) [inline team-norms/retrospectives.md]: (And also define what we mean by action points).

File: team-norms/retrospectives.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Teams are expected to elect a facilitator for their individual retrospectives.
+
+The facilitation role should be seen as a skill that can be improved, and facilitators should treat it as such.
+
+**[Unsolved Problem]** There is value in the facilitator of the comradrospective not becoming a static role. The company needs a way to self-manage who takes this role, either changing every two-weekly or changing every 2,3,4 retrospectives??
+
+## Notes
+
+The format is expected to evolutionary, this document serves as living documentation.
```
Comment (alex-min) [inline team-norms/retrospectives.md]: evolutionary -> evolve

File: README.md [hunk 4]
```diff
@@ -52,3 +52,8 @@ Process
 * [Last Day](security/last_day.md)
 * [Business Development](process/bizdev.md)
 
+Team norms
+--
+
+* [Retrospectives](term-norms/retrospectives.md)
```
Comment (SebAshton) [inline README.md]: `team-norms/retrospectives.md`

File: README.md [hunk 5]
```diff
@@ -52,3 +52,8 @@ Process
 * [Last Day](security/last_day.md)
 * [Business Development](process/bizdev.md)
 
+Team norms
+--
+
+* [Retrospectives](team-norms/retrospectives.md)
+* [Continuous Feedback](policies/continuous_feedback.md)
```
Comment (RoryMacDonald) [inline README.md]: Should we remove 'Continuous Feedback' from policies? It's duplicated now

Comment (craigjbass) [inline README.md]: Potentially, I was hoping that we could move away from some of the traditional language 😄

File: README.md [hunk 6]
```diff
@@ -52,3 +52,8 @@ Process
 * [Last Day](security/last_day.md)
 * [Business Development](process/bizdev.md)
 
+Team norms
+--
+
+* [Retrospectives](team-norms/retrospectives.md)
```
Comment (lukemorton) [inline README.md]: Probably best to put retrospectives in process.

File: README.md (hunk 5 above)
Comment (RoryMacDonald) [inline README.md]: 👍

File: team-norms/retrospectives.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+### Tackling problems
+
+Sometimes it is not immediately obvious how to solve a problem that the team is collectively facing. In these situations it may be wise to have the team attempt to resolve a particularly pressing pain point during normal working time.
+
+Similar to action points, it’s important that the team is able to measure and sense check whether it has made progress towards some goal. 
+
+Make sure the team has awareness on what the outcome of potential solutions would look like.
+
+## Format
```
Comment (RoryMacDonald) [inline team-norms/retrospectives.md]: How are we determining the topics covered in the retrospective / comradrospectives?

File: team-norms/retrospectives.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
+# Retrospectives
+
+These retrospectives occur every 2 weeks on a Friday. There are 2 hours booked for this purpose.
```
Comment (chrisblackburn) [inline team-norms/retrospectives.md]: Ongoing worth considering whether loading 2 hours on a Friday afternoon is the best time?

With dojo and team lunch already, as well as trying to leave the week on a clean slate, we've historically been mindful of pushing too much additional extra-curricular in to Fridays

Comment (craigjbass) [inline team-norms/retrospectives.md]: Given that retrospectives are to reflect on work done, it feels appropriate to do them at the end of a clean slate.

What do you propose?

Comment (craigjbass) [inline team-norms/retrospectives.md]: I think there is a large block of time i.e. 9 days, then a slightly overloaded day. I believe it should be possible for people to work their commitments around this

**Comment by craigjbass** [comment]
Any further feedback will be dealt with in future PR's

**Pull request merged**